docs/source/development_js.rst

```diff
@@ -70,10 +68,14 @@ from within JavaScript.
     $ python setup.py bdist_wheel
 
 
-6. You can now test the ``wheel`` and the ``sdist`` locally before uploading to PyPI.
+5. You can now test the ``wheel`` and the ``sdist`` locally before uploading to PyPI.
 Make sure to use `twine <https://github.com/pypa/twine>`_ to upload the archives over SSL.
 
 .. code::
 
     $ twine upload dist/*
 
+6. If all went well, change the ``notebook/_version.py`` back adding the `.dev` suffix.
+
+7. Push directly on master, not forgetting to push `--tags`.
```

Comment:
Do you need double backticks here and on point 6? I don't think we've done the Sphinx config trick in this repo to make single backticks work like double.